packages/react-examples/src/react/Slider/Slider.Ranged.Example.tsx

```diff
@@ -9,7 +9,7 @@ const sliderValueFormat = (value: number) => `${value}%`;
 export const SliderRangedExample: React.FunctionComponent = () => {
   const [sliderValue, setSliderValue] = React.useState(0);
   const [sliderLowerValue, setSliderLowerValue] = React.useState(0);
-  const onRangeChange = (range: [number, number]) => {
+  const onChange = (_: any, range: [number, number]) => {
```

Comment:
Avoid using `any`. You could use `unknown`, or just `number` since the type is simple.
```suggestion
  const onChange = (_: unknown, range: [number, number]) => {
```